CONTRIBUTING.md

```diff
@@ -45,10 +45,14 @@ cat measures/2018/measures-data.json  | node scripts/validate-data.js measures 2
 
 To create a new perfomance year for measures, run `npm run init:measures $YEAR`. This will create all the necessary folders and files for the new year, as well as increment the quality eMeasureIds and remove last year's spec links from the new measures-data file.
 
-New measures and updates to old measures are handled the same as each other. A CSV file with the proposed changes should be placed in the updates/measures/$YEAR folder with the file name as MM-DD-YYYY-01.csv (increment the final number in the name if multiple changes take place on the same day). IMPORTANT: Do *not* manually modify the Changelog.json, this is updated automatically during the ingestion process. Once the update file is added, run `npm run update:measures $YEAR`. Errors during ingestion will be logged to your terminal, if any.
+New measures and updates to old measures are handled the same as each other. A CSV file with the proposed changes should be placed in the updates/measures/$YEAR folder with the file name as MM-DD-YYYY-01.csv (increment the final number in the name if multiple changes take place on the same day). IMPORTANT: Do *not* manually modify the changes.meta.json, this is updated automatically during the ingestion process. Once the update file is added, run `npm run update:measures $YEAR`. Errors during ingestion will be logged to your terminal, if any.
```

Comment:
I changed the changelog filename to emphasize that it shouldn't be manually changed (unless for good reason) and isn't the source of truth for changes to the measures data (as an actual changelog would be).